src/index.js

```diff
@@ -1,12 +1,12 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import "./index.html"
```

Comment:
I think it is better to import your own module(s) after React modules (just like the original code).

Reply:
If you have an order preference for imports, I can follow that. Afaik, the order doesn't affect any functionality, but in my personal projects I do try to stick to a general order. I do usually put React and third party imports above my own. However in this case, it's the main entry point of the app, `index.html` and I think it makes sense to have it right at the top of the main javascript entry point `index.js`.

Reply:
If you don't mind, please always put the React library modules before your own. It is a style rule that we have been using in all Python and JS code.

Reply:
Is that for all third party imports or just `React`? Any Greenelab style guides or references you could point me to before I start this project would be helpful.